.github/license-lint-config.yml

```diff
@@ -0,0 +1,68 @@
+unrestricted_licenses:
+  - Apache-2.0
+  - BSD-2-Clause
+  - BSD-3-Clause
+  - 0BSD
+  - MIT
+
+restricted_licenses:
```

Comment:
I got them from https://github.com/cncf/foundation/blob/main/allowed-third-party-license-policy.md + the istio config at https://github.com/istio/tools/blob/master/common/config/license-lint.yml#L40

Reply:
What do you mean by "+"? How were they combined?

Reply:
I meant to use "+" as a "set union". So all the CNCF not approved licenses and all the licenses that were in the restricted config in the istio tool